checker/build.gradle

```diff
@@ -28,11 +28,52 @@ jar {
     }
 }
 
+task downloadJdk() {
+    description 'Downloads jdk8.jar and copies it to checker/dist'
+    // See instructions for updating the jdk.jar at https://github.com/typetools/annotated-libraries/blob/master/README.md
+    String shaHash = '3e57e020050ee8b679622e7f864e7dff17aa8acc'
+    File jdk = new File(buildDir, "jdk8-${shaHash}.jar");
+    onlyIf {
+        !rootProject.hasProperty("builtJdk") && (!jdk.exists() || !(new File('"dist/jdk8.jar').exists()))
```

Comment:
Somewhere, please document the meaning of `builtJdk`:
If set (regardless of what value it is set to), then use `dist/jdk8.jar` instead of downloading.